patches/sourcelink/0001-Exclude-test-projects.patch

```diff
@@ -0,0 +1,115 @@
+From 4a051438d29bd96ede45069031462522313ed1bc Mon Sep 17 00:00:00 2001
+From: adaggarwal <aditya.aggarwal@microsoft.com>
+Date: Tue, 2 Jul 2019 16:43:39 +0000
+Subject: [PATCH] Exclude test projects.
+
```

Comment:
I'm not sure why this is needed. SourceLink uses the latest Arcade.

Reply:
Sure, when SourceLink builds standalone and pulls its own Arcade SDK off the network, but source-build doesn't use the latest. (Or at least it didn't, as of writing.) This may also not be the latest SourceLink due to coherency issues, but maybe not since it's newly introduced.

This patch would be classified as a source-build infra issue.